go/pull_request.go

```diff
@@ -628,30 +629,54 @@ func (h *PullRequestHandler) createCobraDocsPreviewPR(
 		return nil, errors.Wrapf(err, "Failed to checkout %s:%s to %s for %s", remote, ref, op, pr.GetHTMLURL())
 	}
 
+	var (
+		tree            *github.Tree
+		commit          *github.Commit
+		skipFirstCommit bool
+	)
+
 	// 3. Run the sync script with `COBRADOC_VERSION_PAIRS="$(baseref):$(docsVersion)`.
 	_, err = shell.NewContext(ctx, "./tools/sync_cobradocs.sh").InDir(website.LocalDir).WithExtraEnv(
 		fmt.Sprintf("VITESS_DIR=%s", vitess.LocalDir),
 		"COBRADOCS_SYNC_PERSIST=yes",
 		fmt.Sprintf("COBRADOC_VERSION_PAIRS=HEAD:%s", docsVersion),
 	).Output()
 	if err != nil {
-		return nil, errors.Wrapf(err, "Failed to run cobradocs sync script against %s:%s to %s for %s", remote, ref, op, pr.GetHTMLURL())
+		var exitErr *exec.ExitError
+		if (errors.As(err, &exitErr)) &&
+			(bytes.Contains(exitErr.Stderr, []byte("No changes to cobradocs detected"))) {
```

Comment:
nit picking but there are un-required parenthesis on this if condition
```suggestion
		if errors.As(err, &exitErr) &&
			bytes.Contains(exitErr.Stderr, []byte("No changes to cobradocs detected")) {
```